agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -373,6 +393,8 @@ private static String getDefaultPath() {
   }
 
   public static class SamplingOverride {
+    // TODO (trask) consider making this required when moving out of preview
+    @Nullable public SpanKind spanKind;
```

Comment:
declare it as io.opentelemetry.api.trace.SpanKind?

Reply:
see comment above